components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -360,6 +373,20 @@ subroutine crm_physics_init(state, pbuf2d, species_class)
 
    ! Initialize pbuf variables
    if (is_first_step()) then
+
+      call pbuf_set_field(pbuf2d, crm_t_rad_idx,  0._r8)
+      call pbuf_set_field(pbuf2d, crm_qv_rad_idx, 0._r8)
+      call pbuf_set_field(pbuf2d, crm_qc_rad_idx, 0._r8)
+      call pbuf_set_field(pbuf2d, crm_qi_rad_idx, 0._r8)
+      call pbuf_set_field(pbuf2d, crm_cld_rad_idx,0._r8)
+      call pbuf_set_field(pbuf2d, crm_qrad_idx,   0._r8)
+      if (MMF_microphysics_scheme .eq. 'm2005') then
+         call pbuf_set_field(pbuf2d, crm_rad_idx_nc,0._r8)
```

Comment:
Should be `crm_nc_rad_idx` consistent with above.

Reply:
fixed